ocaml/idl/datamodel.ml

```diff
@@ -1335,8 +1335,10 @@ let _ =
 		~doc:"The VM cannot be imported unforced because it is either the same version or an older version of an existing VM." ();
 
 	error Api_errors.vm_call_plugin_rate_limit ["VM"; "interval"; "wait"]
-		~doc:"There is a minimal interval required between consecutive plugin calls made on the same VM, please wait before retry." ()
+		~doc:"There is a minimal interval required between consecutive plugin calls made on the same VM, please wait before retry." ();
 
+	error Api_errors.vm_cant_migrate ["VM"]
+		~doc:"The VM can't migrate because it is either explicitly blocked or it uses nested virtualisation"
```

Comment:
This should be more generic, as migration is only one of the operations blocked.  How about `vm_not_mobile`, which is a term used elsewhere?

The text should say something like "The VM is configured in a way which prevents it being mobile".  This also includes situations like having a PCI device attached.  It should not mention nested virt.